[2024-10-18T11:59:40.448+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-18T11:59:40.530+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.this_is_1st_task scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2024-10-18T11:59:40.575+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.this_is_1st_task scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2024-10-18T11:59:40.579+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-18T11:59:40.717+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): this_is_1st_task> on 2022-01-08 00:00:00+00:00
[2024-10-18T11:59:40.783+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=884) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-18T11:59:40.786+0000] {standard_task_runner.py:72} INFO - Started process 894 to run task
[2024-10-18T11:59:40.771+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'this_is_1st_task', 'scheduled__2022-01-08T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp7t2djk8t']
[2024-10-18T11:59:40.801+0000] {standard_task_runner.py:105} INFO - Job 169: Subtask this_is_1st_task
[2024-10-18T11:59:41.091+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.this_is_1st_task scheduled__2022-01-08T00:00:00+00:00 [running]> on host 8e48fbd237c9
[2024-10-18T11:59:41.644+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sourabh' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='this_is_1st_task' AIRFLOW_CTX_EXECUTION_DATE='2022-01-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-08T00:00:00+00:00'
[2024-10-18T11:59:41.667+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-18T11:59:41.883+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-18T11:59:41.886+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is my first task']
[2024-10-18T11:59:41.970+0000] {subprocess.py:86} INFO - Output:
[2024-10-18T11:59:41.972+0000] {subprocess.py:93} INFO - Hello world, this is my first task
[2024-10-18T11:59:41.974+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-18T11:59:42.118+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-18T11:59:42.124+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=this_is_1st_task, run_id=scheduled__2022-01-08T00:00:00+00:00, execution_date=20220108T000000, start_date=20241018T115940, end_date=20241018T115942
[2024-10-18T11:59:42.226+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-18T11:59:42.287+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-18T11:59:42.337+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
